fast_td3/evaluation.ipynb

```
@@ -262,51 +248,125 @@
-      "text/plain": [
-       "(375.83074951171875, 548.75)"
-      ]
-     },
-     "execution_count": 37,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
+   "source": [
+    "import tempfile\n",
```

Pylint (4.0.2)

[error] Undefined variable 'jnp'

(E0602)